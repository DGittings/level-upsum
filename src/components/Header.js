import React from 'react';
import styled from 'styled-components';

const MainHeader = styled.header`
  text-align: center;
  padding: 150px 0 75px;

  > div {
    max-width: 800px;
    margin: 0 auto;
  }
`;

const Title = styled.h1`
  margin-top: 0;
  font-size: 50px;
`;

const Subheading = styled.h3`
  font-size: 28px;
`;

const Header = () => (
  <MainHeader>
    <div>
      <Title>Gamer ipsum</Title>
      <Subheading>Placeholder text for gamer types</Subheading>
    </div>
  </MainHeader>
);

export default Header;
