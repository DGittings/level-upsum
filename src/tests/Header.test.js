import React from 'react';
import Header from '../components/Header';
import renderer from 'react-test-renderer';

test('Test component snapshot', () => {
  const component = renderer.create(<Header />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
