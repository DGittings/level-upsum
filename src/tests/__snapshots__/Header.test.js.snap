// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component snapshot 1`] = `
<header
  className="sc-bdVaJa jGQOhP"
>
  <div>
    <h1
      className="sc-bwzfXH hRNNch"
    >
      Level upsum
    </h1>
    <h3
      className="sc-htpNat etFLsn"
    >
      Video game placeholder text
    </h3>
  </div>
</header>
`;
